packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/ai-agent-action/components/AIChatTab.tsx

```diff
@@ -308,6 +327,28 @@ export const AIChatTab = ({ agentId }: { agentId: string }) => {
           onChange={handleInputChange}
         />
         <StyledButtonsContainer>
+          {!isWorkflowAgentNodeChat && (
```

Comment:
FYI Thomas was arguing that having chat in workflow was confusing and we should remove the Chat tab in workflows and replace it with a non-chat mode where a chat message would erase the previous response. I think the box could be prefilled with the "Instructions for AI" or something similar. No need to to it in this PR but telling you FYI.